R/MoBiProject.R

```diff
@@ -11,36 +11,61 @@ MoBiProject <- R6::R6Class(
     sourceFile = function(value) {
       private$.readOnlyProperty("sourceFile", value, private$.sourceFile)
     },
-    #' @field simulationNames Names of the simulations that are present in the project
+    #' @field simulationNames Names of the simulations that are present in the project (read-only)
     simulationNames = function(value) {
       if (missing(value)) {
-        model <- self$get("SimulationNames")
+        netTask <- .getNetTaskFromCache("ProjectTask", isMoBiR = TRUE)
+        values <- netTask$call("AllSimulationNames", self)
+        return(values$call("ToArray"))
       } else {
         private$.throwPropertyIsReadonly("simulationNames")
       }
     },
-    #' @field parameterIdentificationNames Names of the parameter identifications that are present in the project
+
+    #' @field moduleNames Names of the modules that are present in the project (read-only)
+    moduleNames = function(value) {
+      if (missing(value)) {
+        netTask <- .getNetTaskFromCache("ProjectTask", isMoBiR = TRUE)
+        values <- netTask$call("AllModuleNames", self)
```

Comment:
Encapsulate these 2 lines of code in something like `getProjectProperty(propertyName)`?

Reply:
Implemented as `callProjectTaskAsArray`